spec/graphql/types/organization_type_spec.rb

```diff
@@ -8,6 +8,10 @@
   let!(:user3) { create :user }
   let!(:user4) { create :user }
   let!(:organization) { create :organization }
+  let!(:repository1) { create(:repository_compound, owner: organization) }
+  let!(:repository2) { create(:repository_compound, owner: organization) }
+  let!(:repository3) { create(:repository_compound, owner: organization) }
+  let!(:repository4) { create(:repository_compound) }
```

Comment:
Unless you want to use these variables in the test code below, I suggest to clean the code by moving this into a before block:
```ruby
before do
  3.times { create(:repository_compound, owner: organization) }
  create(:repository_compound)
end
```